docs/proposals/scheduledjob.md

```diff
@@ -216,6 +210,151 @@ In the above example:
   July 7th, 2pm.  This value will be validated according to the same rules which
   apply to `.spec.schedule`.
 
+## Fields Added to Job Template
+
+When the controller creates a Job from the JobTemplateSpec in the ScheduledJob, it
+adds the following fields to the Job:
+
+- a name, based on the ScheduledJob's name, but with a suffix to distinguish
+  multiple executions, which may overlap.
+- the standard created-by annotation on the Job, pointing to the SJ that created it
+  The standard key is `kubernetes.io/created-by`.  The value is a serialized JSON object, like
+  `{ "kind":"SerializedReference","apiVersion":"v1","reference":{"kind":"ScheduledJob","namespace":"default",`
+  `"name":"nightly-earnings-report","uid":"5ef034e0-1890-11e6-8935-42010af0003e","apiVersion":...`
+  This serialization contains the UID of the parent.  This is used to match the Job to the SJ that created
+  it.
+
+## Updates to ScheduledJobs
+
+If the schedule is updated on a ScheduledJob, it will:
+- continue to use the Status.Active list of jobs to detect conflicts.
+- continue to use the Status.Active list of jobs as fulfilling the schedule if they exactly match
+  a start time on the current schedule.
+  - Example:   If you have a schedule to run every 30 minutes, and change that to hourly, then the previously started
+    top-of-the-hour run, in Status.Active, will be seen and no new job starte.
+  - Example 2: If you have a schedule of daily at 10am, and then at 11am you update
+    it to run daily at 9:55am, then the preceding 10am run will not count as fulfilling the requirement to run
+    at 9:55, and a new job may be launched (if it is not past the starting deadline).
```

Comment:
Hmmm.... At 11am, before the update the run was accounted as correct, if it finished. If it's still running it should not be started but counted as well. 

Generally whatever started and finished will be counted in Active, imho. I don't think we'll be modifying those in any way.